core/log.go

```diff
@@ -66,7 +67,24 @@ func (l *Logger) Log(level int, format string, args ...interface{}) {
 		payload := fmt.Sprintf(session.Config.WebhookPayload, text)
 		http.Post(session.Config.Webhook, "application/json", strings.NewReader(payload))
 	}
-
+	
+	if session.Config.Logstash != "" {
+		text := colorStrip(fmt.Sprintf(format, args...))
+		payload := fmt.Sprintf(session.Config.Logstash, text)
+  		pc, err := net.ListenPacket("udp4", ":" + session.Config.LogstashPort)
+  		if err != nil {
+    		panic(err)
```

Comment:
I don't think it should panic out "just" because the logstash session has an error.

Reply:
That's fair, I can get those removed and pushed to the branch.